protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -157,6 +157,23 @@ module atomic_bridge::atomic_bridge_counterparty {
             },
         );
     }
+
+    public entry fun noop_for_testing(_signer: &signer) {
```

Comment:
is this needed? maybe #[test_only] if so and a comment why

Reply:
Removed it.